src/commissaire/containermgr/trivial.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2017  Red Hat, Inc
```

Comment:
Since this is only ever to be used in testing could we move this into the ``test/`` directory?